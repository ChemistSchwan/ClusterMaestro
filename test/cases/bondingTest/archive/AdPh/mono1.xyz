66
Properties=species:S:1:pos:R:3:tags:I:1 pbc="F F F"
C       1.07573747       0.69576201       4.19532208        0 
C      -0.26372455       0.27200459       4.81764897        1 
C      -0.13099707       1.93596400       2.38691879        2 
C      -1.48331691       1.53455830       2.98490771        3 
C      -1.93108409       0.19693768       2.38678173        4 
C      -0.89990223      -0.89922013       2.67835150        5 
C       0.44990576      -0.46639674       2.09328051        6 
C       0.92894555       0.86762289       2.67853999        7 
C      -0.73335002      -1.05199111       4.19513594        8 
C      -1.30631352       1.35122178       4.49712722        9 
H      -0.20633076       2.06426232       1.30747517       10 
H       0.16641597       2.89685294       2.80871108       11 
H      -2.05667340       0.27675801       1.30734622       12 
H      -2.90166738      -0.06721962       2.80852982       13 
H       1.19205481      -1.23458332       2.32066014       14 
H       0.35501820      -0.36811874       1.00921399       15 
H      -1.67983269      -1.34616913       4.65326529       16 
H      -0.01250342      -1.84635122       4.39604166       17 
H       1.40234997       1.63148091       4.65359747       18 
H       1.84473526      -0.05208441       4.39620063       19 
H      -0.98348761       2.29821617       4.93507756       20 
H      -2.26390446       1.06119557       4.93497850       21 
C      -0.07390949       0.07540704       6.31440102       22 
C       0.84154931      -0.87220168       6.76763993       23 
C      -0.77396106       0.79995011       7.27153648       24 
C      -0.56860108       0.58729784       8.62480276       25 
C       1.04963299      -1.08767535       8.11648606       26 
C       0.34354208      -0.35686541       9.05604877       27 
H      -1.49512014       1.54643349       6.98048154       28 
H       1.40832819      -1.45878812       6.05989858       29 
H      -1.12855853       1.16682550       9.34474580       30 
H       1.76691737      -1.83013385       8.43547359       31 
H       0.50334344      -0.52237087      10.11113123       32 
C      -2.51265052       2.60013815       2.63983727       33 
C      -3.21272725       3.32454362       3.59656913       34 
C      -2.78079450       2.87821383       1.30140568       35 
C      -3.70493150       3.83517947       0.93019480       36 
C      -4.14099357       4.28575959       3.23039767       37 
C      -4.39308649       4.54731621       1.89707401       38 
H      -2.25934473       2.33854807       0.52569863       39 
H      -3.04745657       3.15303352       4.64796978       40 
H      -3.88861411       4.02575626      -0.11752486       41 
H      -4.66976182       4.83291784       3.99759820       42 
H      -5.11674181       5.29673349       1.61289571       43 
C       2.25204932       1.31832939       2.07555921       44 
C       2.97960546       0.54184138       1.18274801       45 
C       2.77151157       2.56739436       2.40689193       46 
C       3.94582095       3.03157370       1.84635852       47 
C       4.15403328       1.00332196       0.61336634       48 
C       4.63973027       2.25551842       0.93504563       49 
H       2.25717502       3.19695239       3.11749531       50 
H       2.64824653      -0.44952188       0.92059051       51 
H       4.32004148       4.00688234       2.12185931       52 
H       4.68764329       0.37466829      -0.08507702       53 
H       5.55228678       2.62038213       0.48812292       54 
C      -1.39580586      -2.20600078       2.07523477       55 
C      -0.64461137      -2.95989117       1.18262245       56 
C      -2.66208531      -2.68215594       2.40628729       57 
C      -3.16628293      -3.83980898       1.84567419       58 
C      -1.14610745      -4.11776870       0.61317457       59 
C      -2.41435765      -4.56006362       0.93457554       60 
H      -3.27375959      -2.14640744       3.11669091       61 
H       0.35761353      -2.66285546       0.92062315       62 
H      -4.15398542      -4.18019590       2.12090805       63 
H      -0.53604333      -4.67269595      -0.08514458       64 
H      -2.81031624      -5.45952957       0.48757719       65 
