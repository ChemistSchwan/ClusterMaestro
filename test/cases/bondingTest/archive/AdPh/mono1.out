      -----------------------------------------------------------      
     |                   =====================                   |     
     |                           x T B                           |     
     |                   =====================                   |     
     |                         S. Grimme                         |     
     |          Mulliken Center for Theoretical Chemistry        |     
     |                    University of Bonn                     |     
      -----------------------------------------------------------      

   * xtb version 6.2.3 (830e466) compiled by 'ehlert@majestix' on 2020-03-16

   xtb is free software: you can redistribute it and/or modify it under
   the terms of the GNU Lesser General Public License as published by
   the Free Software Foundation, either version 3 of the License, or
   (at your option) any later version.
   
   xtb is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU Lesser General Public License for more details.
   
   Cite this work as:
   * S. Grimme, C. Bannwarth, P. Shushkov, J. Chem. Theory Comput., 2017,
     13, 1989-2009. DOI: 10.1021/acs.jctc.7b00118
   * C. Bannwarth, S. Ehlert and S. Grimme., J. Chem. Theory Comput., 2019,
     15, 1652-1671. DOI: 10.1021/acs.jctc.8b01176
   * P. Pracht, E. Caldeweyher, S. Ehlert, S. Grimme, ChemRxiv, 2019, preprint.
     DOI: 10.26434/chemrxiv.8326202.v1
   
   for DFT-D4:
   * E. Caldeweyher, C. Bannwarth and S. Grimme, J. Chem. Phys., 2017,
     147, 034112. DOI: 10.1063/1.4993215
   * E. Caldeweyher, S. Ehlert, A. Hansen, H. Neugebauer, S. Spicher,
     C. Bannwarth and S. Grimme, J. Chem. Phys., 2019, 150, 154122.
     DOI: 10.1063/1.5090222
   
   for sTDA-xTB:
   * S. Grimme and C. Bannwarth, J. Chem. Phys., 2016, 145, 054103.
     DOI: 10.1063/1.4959605
   
   in the mass-spec context:
   * V. Asgeirsson, C. Bauer and S. Grimme, Chem. Sci., 2017, 8, 4879.
     DOI: 10.1039/c7sc00601b
   
   for metadynamics refer to:
   * S. Grimme, J. Chem. Theory Comput., 2019, 155, 2847-2862
     DOI: 10.1021/acs.jctc.9b00143
   
   with help from (in alphabetical order)
   C. Bannwarth, F. Bohle, G. Brandenburg, E. Caldeweyher, M. Checinski,
   S. Dohm, S. Ehlert, S. Ehrlich, F. MÃ¤rz, H. Neugebauer, J. Pisarek,
   P. Pracht, P. Shushkov, and S. Spicher.
   
 * started run on 2020/09/25 at 10:01:14.769     

           -------------------------------------------------
          |                Calculation Setup                |
           -------------------------------------------------

          program call               : xtb start.xyz -P 1
          coordinate file            : start.xyz
          omp threads                :                     1
          number of atoms            :                    66
          number of electrons        :                   168
          charge                     :                     0
          spin                       :                   0.0
          first test random number   :      0.24569138413471

########################################################################
[WARNING] Please study the warnings concerning your input carefully
-2- prog_main: XTBPATH is not set, using XTBHOME instead
-1- prog_main: XTBHOME is not set, using HOME instead
########################################################################
           ------------------------------------------------- 
          |                 G F N 2 - x T B                 |
          | Geometry, Frequencies, Noncovalent interactions |
          |            JCTC 2019 parametrisation            |
           ------------------------------------------------- 
             k(s)              :                1.8500
             k(p)              :                2.2300
             k(d)              :                2.2300
             k(f)              :                2.0000
             kEN (H0ij)        :               -2.0000
             D4 a1             :                0.5200
             D4 a2             :                5.0000
             D4 s6             :                1.0000
             D4 s8             :                2.7000
             D4 s9             :                5.0000
             alphaj            :                2.0000

  Z AO/shell   Hii/eV     exponent
  1     Wed Apr 25 08:07:45 CEST 2018   EN: 2.200 GM2: 0.406  GM3: 0.0800  RAES: 1.40
     1s    -10.707211    1.230000
  6     Wed Apr 25 08:07:45 CEST 2018   EN: 2.550 GM2: 0.538  GM3: 0.1500  RAES: 3.00
     2s    -13.970922    2.096432
     2p    -10.063292    1.800000

           ------------------------------------------------- 
          |        Self-Consistent Charge Iterations        |
           ------------------------------------------------- 

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :  # basis functions                 168          :
          :  # atomic orbitals                 168          :
          :  # shells                          100          :
          :  # electrons                       168          :
          :  max. iterations                   250          :
          :  Hamiltonian                  GFN2-xTB          :
          :  restarted?                      false          :
          :  GBSA solvation                  false          :
          :  PC potential                    false          :
          :  electronic temp.          300.0000000     K    :
          :  accuracy                    1.0000000          :
          :  -> integral cutoff          0.2500000E+02      :
          :  -> integral neglect         0.1000000E-07      :
          :  -> SCF convergence          0.1000000E-05 Eh   :
          :  -> wf. convergence          0.1000000E-03 e    :
          :  Broyden damping             0.4000000          :
          ...................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1    -90.6973885 -0.906974E+02  0.344E+00    4.51       0.0  T
   2    -90.7604594 -0.630710E-01  0.203E+00    4.49       1.0  T
   3    -90.7614156 -0.956206E-03  0.500E-01    4.44       1.0  T
   4    -90.7630700 -0.165432E-02  0.206E-01    4.45       1.0  T
   5    -90.7650299 -0.195994E-02  0.469E-02    4.46       1.0  T
   6    -90.7650642 -0.343371E-04  0.253E-02    4.45       1.0  T
   7    -90.7650693 -0.507883E-05  0.517E-03    4.46       2.4  T
   8    -90.7650700 -0.637815E-06  0.162E-03    4.46       7.6  T
   9    -90.7650700 -0.453653E-07  0.926E-04    4.46      13.3  T
  10    -90.7650700 -0.271741E-07  0.215E-04    4.46      57.2  T

   *** convergence criteria satisfied after 10 iterations ***

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.6535240             -17.7833
       ...           ...                  ...                  ...
        78        2.0000           -0.4001043             -10.8874
        79        2.0000           -0.4000192             -10.8851
        80        2.0000           -0.3992821             -10.8650
        81        2.0000           -0.3916500             -10.6573
        82        2.0000           -0.3883775             -10.5683
        83        2.0000           -0.3883252             -10.5669
        84        2.0000           -0.3872177             -10.5367 (HOMO)
        85                         -0.2234910              -6.0815 (LUMO)
        86                         -0.2192048              -5.9649
        87                         -0.2191960              -5.9646
        88                         -0.2186898              -5.9509
        89                         -0.2183645              -5.9420
       ...                                ...                  ...
       168                          0.6705322              18.2461
      -------------------------------------------------------------
                  HL-Gap            0.1637266 Eh            4.4552 eV
             Fermi-level           -0.3053544 Eh           -8.3091 eV

 SCC (total)                   0 d,  0 h,  0 min,  0.226 sec
 SCC setup                      ...        0 min,  0.001 sec (  0.409%)
 Dispersion                     ...        0 min,  0.002 sec (  0.879%)
 integral evaluation            ...        0 min,  0.087 sec ( 38.295%)
 zeroth order Hamiltonian       ...        0 min,  0.001 sec (  0.333%)
 iterations                     ...        0 min,  0.074 sec ( 32.785%)
 molecular gradient             ...        0 min,  0.061 sec ( 26.789%)
 printout                       ...        0 min,  0.001 sec (  0.492%)

         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         ::                     SUMMARY                     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         :: total energy             -89.156625036890 Eh    ::
         :: gradient norm              0.004257841556 Eh/a0 ::
         :: HOMO-LUMO gap              4.455228837678 eV    ::
         ::.................................................::
         :: SCC energy               -90.765070023740 Eh    ::
         :: -> isotropic ES            0.005222470662 Eh    ::
         :: -> anisotropic ES          0.014949919941 Eh    ::
         :: -> anisotropic XC          0.053865601910 Eh    ::
         :: -> dispersion             -0.087091396957 Eh    ::
         :: repulsion energy           1.606876920018 Eh    ::
         :: add. restraining           0.000000000000 Eh    ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::

           -------------------------------------------------
          |                Property Printout                |
           -------------------------------------------------

    * Orbital Energies and Occupations

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.6535240             -17.7833
       ...           ...                  ...                  ...
        72        2.0000           -0.4169977             -11.3471
        73        2.0000           -0.4105852             -11.1726
        74        2.0000           -0.4064280             -11.0595
        75        2.0000           -0.4062599             -11.0549
        76        2.0000           -0.4001845             -10.8896
        77        2.0000           -0.4001557             -10.8888
        78        2.0000           -0.4001043             -10.8874
        79        2.0000           -0.4000192             -10.8851
        80        2.0000           -0.3992821             -10.8650
        81        2.0000           -0.3916500             -10.6573
        82        2.0000           -0.3883775             -10.5683
        83        2.0000           -0.3883252             -10.5669
        84        2.0000           -0.3872177             -10.5367 (HOMO)
        85                         -0.2234910              -6.0815 (LUMO)
        86                         -0.2192048              -5.9649
        87                         -0.2191960              -5.9646
        88                         -0.2186898              -5.9509
        89                         -0.2183645              -5.9420
        90                         -0.2180529              -5.9335
        91                         -0.2180219              -5.9327
        92                         -0.2146760              -5.8416
        93                         -0.0931086              -2.5336
        94                         -0.0902500              -2.4558
        95                         -0.0901856              -2.4541
       ...                                ...                  ...
       168                          0.6705322              18.2461
      -------------------------------------------------------------
                  HL-Gap            0.1637266 Eh            4.4552 eV
             Fermi-level           -0.3053544 Eh           -8.3091 eV

     #   Z        covCN         q      C6AA      Î±(0)
     1   6 C      3.822    -0.055    21.546     6.605
     2   6 C      3.910     0.016    20.243     6.395
     3   6 C      3.825    -0.056    21.565     6.608
     4   6 C      3.911     0.017    20.239     6.394
     5   6 C      3.825    -0.056    21.565     6.608
     6   6 C      3.910     0.016    20.243     6.395
     7   6 C      3.821    -0.060    21.635     6.619
     8   6 C      3.910     0.016    20.243     6.395
     9   6 C      3.822    -0.055    21.546     6.605
    10   6 C      3.820    -0.060    21.635     6.619
    11   1 H      0.925     0.031     2.569     2.506
    12   1 H      0.924     0.031     2.566     2.505
    13   1 H      0.925     0.031     2.569     2.506
    14   1 H      0.924     0.031     2.566     2.505
    15   1 H      0.924     0.033     2.543     2.494
    16   1 H      0.924     0.033     2.550     2.497
    17   1 H      0.924     0.030     2.584     2.514
    18   1 H      0.924     0.031     2.573     2.508
    19   1 H      0.924     0.030     2.584     2.514
    20   1 H      0.924     0.031     2.573     2.508
    21   1 H      0.924     0.033     2.546     2.495
    22   1 H      0.924     0.033     2.546     2.495
    23   6 C      2.973     0.005    27.902     8.665
    24   6 C      2.928    -0.036    28.890     8.815
    25   6 C      2.926    -0.039    28.979     8.828
    26   6 C      2.920    -0.031    28.784     8.798
    27   6 C      2.919    -0.031    28.775     8.797
    28   6 C      2.919    -0.033    28.826     8.804
    29   1 H      0.926     0.031     2.572     2.508
    30   1 H      0.926     0.027     2.634     2.538
    31   1 H      0.926     0.028     2.621     2.532
    32   1 H      0.926     0.028     2.621     2.532
    33   1 H      0.926     0.028     2.621     2.532
    34   6 C      2.973     0.005    27.911     8.666
    35   6 C      2.926    -0.039    28.978     8.828
    36   6 C      2.929    -0.036    28.892     8.815
    37   6 C      2.919    -0.031    28.773     8.796
    38   6 C      2.920    -0.032    28.786     8.798
    39   6 C      2.920    -0.033    28.826     8.804
    40   1 H      0.926     0.027     2.632     2.537
    41   1 H      0.926     0.031     2.577     2.510
    42   1 H      0.926     0.028     2.620     2.531
    43   1 H      0.926     0.028     2.621     2.532
    44   1 H      0.926     0.027     2.622     2.532
    45   6 C      2.972     0.005    27.912     8.666
    46   6 C      2.928    -0.039    28.980     8.828
    47   6 C      2.929    -0.036    28.889     8.814
    48   6 C      2.920    -0.031    28.774     8.796
    49   6 C      2.920    -0.031    28.785     8.798
    50   6 C      2.920    -0.033    28.822     8.804
    51   1 H      0.926     0.027     2.633     2.538
    52   1 H      0.926     0.031     2.575     2.510
    53   1 H      0.926     0.028     2.620     2.531
    54   1 H      0.926     0.028     2.620     2.531
    55   1 H      0.926     0.027     2.622     2.532
    56   6 C      2.972     0.005    27.912     8.666
    57   6 C      2.928    -0.039    28.980     8.828
    58   6 C      2.929    -0.036    28.889     8.814
    59   6 C      2.920    -0.031    28.774     8.796
    60   6 C      2.920    -0.031    28.785     8.798
    61   6 C      2.920    -0.033    28.822     8.804
    62   1 H      0.926     0.027     2.633     2.538
    63   1 H      0.926     0.031     2.575     2.510
    64   1 H      0.926     0.028     2.620     2.531
    65   1 H      0.926     0.028     2.620     2.531
    66   1 H      0.926     0.027     2.622     2.532

 Mol. C6AA /auÂ·bohrâ¶  :      50954.124792
 Mol. C8AA /auÂ·bohrâ¸  :    1245574.136267
 Mol. Î±(0) /au        :        356.690141


Wiberg/Mayer (AO) data.
largest (>0.10) Wiberg bond orders for each atom
          total WBO             WBO to atom ...
     1  C   3.996        C    8 0.974    C    2 0.974    H   19 0.973    H   20 0.972
     2  C   3.989        C   10 0.986    C   23 0.982    C    1 0.974    C    9 0.974
     3  C   3.996        C    8 0.975    C    4 0.974    H   11 0.972    H   12 0.970
     4  C   3.989        C   10 0.986    C   34 0.981    C    5 0.974    C    3 0.974
     5  C   3.996        C    6 0.975    C    4 0.974    H   13 0.972    H   14 0.970
     6  C   3.989        C    7 0.986    C   56 0.981    C    5 0.975    C    9 0.974
     7  C   3.997        C    8 0.986    C    6 0.986    H   16 0.972    H   15 0.969
     8  C   3.989        C    7 0.986    C   45 0.981    C    3 0.975    C    1 0.974
     9  C   3.996        C    6 0.974    C    2 0.974    H   17 0.973    H   18 0.972
    10  C   3.997        C    2 0.986    C    4 0.986    H   21 0.971    H   22 0.971
    11  H   0.999        C    3 0.972
    12  H   0.998        C    3 0.970
    13  H   0.999        C    5 0.972
    14  H   0.998        C    5 0.970
    15  H   0.999        C    7 0.969
    16  H   0.999        C    7 0.972
    17  H   0.999        C    9 0.973
    18  H   0.999        C    9 0.972
    19  H   0.999        C    1 0.973
    20  H   0.999        C    1 0.972
    21  H   0.999        C   10 0.971
    22  H   0.999        C   10 0.971
    23  C   3.984        C   25 1.420    C   24 1.404    C    2 0.982    C   28 0.111
    24  C   3.991        C   27 1.454    C   23 1.404    H   30 0.965    C   26 0.112
    25  C   3.991        C   26 1.437    C   23 1.420    H   29 0.966    C   27 0.109
    26  C   3.992        C   28 1.445    C   25 1.437    H   31 0.971    C   24 0.112
    27  C   3.992        C   24 1.454    C   28 1.431    H   32 0.971    C   25 0.109
    28  C   3.991        C   26 1.445    C   27 1.431    H   33 0.972    C   23 0.111
    29  H   0.999        C   25 0.966
    30  H   0.999        C   24 0.965
    31  H   0.999        C   26 0.971
    32  H   0.999        C   27 0.971
    33  H   0.999        C   28 0.972
    34  C   3.984        C   35 1.421    C   36 1.403    C    4 0.981    C   39 0.111
    35  C   3.991        C   38 1.436    C   34 1.421    H   41 0.966    C   37 0.109
    36  C   3.991        C   37 1.455    C   34 1.403    H   40 0.965    C   38 0.112
    37  C   3.992        C   36 1.455    C   39 1.430    H   42 0.971    C   35 0.109
    38  C   3.992        C   39 1.446    C   35 1.436    H   43 0.971    C   36 0.112
    39  C   3.991        C   38 1.446    C   37 1.430    H   44 0.972    C   34 0.111
    40  H   0.999        C   36 0.965
    41  H   0.999        C   35 0.966
    42  H   0.999        C   37 0.971
    43  H   0.999        C   38 0.971
    44  H   0.999        C   39 0.972
    45  C   3.984        C   46 1.420    C   47 1.404    C    8 0.981    C   50 0.111
    46  C   3.991        C   49 1.438    C   45 1.420    H   52 0.965    C   48 0.109
    47  C   3.991        C   48 1.453    C   45 1.404    H   51 0.965    C   49 0.112
    48  C   3.992        C   47 1.453    C   50 1.431    H   53 0.971    C   46 0.109
    49  C   3.992        C   50 1.445    C   46 1.438    H   54 0.971    C   47 0.112
    50  C   3.991        C   49 1.445    C   48 1.431    H   55 0.971    C   45 0.111
    51  H   0.999        C   47 0.965
    52  H   0.999        C   46 0.965
    53  H   0.999        C   48 0.971
    54  H   0.999        C   49 0.971
    55  H   0.999        C   50 0.971
    56  C   3.984        C   57 1.420    C   58 1.404    C    6 0.981    C   61 0.111
    57  C   3.991        C   60 1.438    C   56 1.420    H   63 0.965    C   59 0.109
    58  C   3.991        C   59 1.453    C   56 1.404    H   62 0.965    C   60 0.112
    59  C   3.992        C   58 1.453    C   61 1.431    H   64 0.971    C   57 0.109
    60  C   3.992        C   61 1.445    C   57 1.438    H   65 0.971    C   58 0.112
    61  C   3.991        C   60 1.445    C   59 1.431    H   66 0.971    C   56 0.111
    62  H   0.999        C   58 0.965
    63  H   0.999        C   57 0.965
    64  H   0.999        C   59 0.971
    65  H   0.999        C   60 0.971
    66  H   0.999        C   61 0.971

molecular dipole:
                 x           y           z       tot (Debye)
 q only:        0.004      -0.005       0.010
   full:        0.006      -0.006       0.009       0.032
molecular quadrupole (traceless):
                xx          xy          yy          xz          yz          zz
 q only:       -0.284      -1.199      -0.202      -1.402       1.451       0.486
  q+dip:       -0.950      -4.565      -0.635      -5.295       5.479       1.585
   full:       -0.464      -2.206      -0.312      -2.542       2.630       0.775


           -------------------------------------------------
          | TOTAL ENERGY              -89.156625036890 Eh   |
          | GRADIENT NORM               0.004257841556 Eh/Î± |
          | HOMO-LUMO GAP               4.455228837678 eV   |
           -------------------------------------------------

------------------------------------------------------------------------
 * finished run on 2020/09/25 at 10:01:15.074     
------------------------------------------------------------------------
 total:
 * wall-time:     0 d,  0 h,  0 min,  0.304 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.291 sec
 * ratio c/w:     0.954 speedup
 SCF:
 * wall-time:     0 d,  0 h,  0 min,  0.226 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.226 sec
 * ratio c/w:     0.999 speedup

